app/editor/src/store/hooks/admin/mediatypes/useMediaTypes.ts

```diff
@@ -0,0 +1,80 @@
+import { IMediaTypeModel, IPaged, useApiAdminMediaTypes } from 'hooks/api-editor';
+import React from 'react';
+import { useApiDispatcher } from 'store/hooks';
+import { IAdminState, useAdminStore } from 'store/slices';
+
+interface IMediaTypeController {
+  findAllMediaTypes: () => Promise<IMediaTypeModel[]>;
+  findMediaTypes: () => Promise<IPaged<IMediaTypeModel>>;
+  getMediaType: (id: number) => Promise<IMediaTypeModel>;
+  addMediaType: (model: IMediaTypeModel) => Promise<IMediaTypeModel>;
+  updateMediaType: (model: IMediaTypeModel) => Promise<IMediaTypeModel>;
+  deleteMediaType: (model: IMediaTypeModel) => Promise<IMediaTypeModel>;
+}
+
+export const useMediaTypes = (): [IAdminState, IMediaTypeController] => {
+  const api = useApiAdminMediaTypes();
+  const dispatch = useApiDispatcher();
+  const [state, store] = useAdminStore();
+
+  const getMediaTypes = () => state.mediaTypes;
```

Comment:
I'm curious what you're trying here?  Based on this warning I suspect it isn't working though.

Reply:
Was just keeping consistent with useDataSources.ts but got rid of getMediaTypes now!